keanu-python/examples/lorenz_model.py

```diff
@@ -17,10 +17,12 @@ def run_model(self, num_time_steps):
         for _ in range(num_time_steps):
             position = self.__get_next_position(position)
             yield position
-        
-    def __get_next_position(self, current : Coordinates) -> Coordinates :
-        nextX = current.x + self.time_step * (self.sigma * (current.y - current.x))
```

Comment:
How long is the max line length? Is it possible to increase it?

Reply:
I'm wondering this too

Reply:
Google style is 80 by default. We can override this. Shall I try 100 and see what that looks like?

Reply:
go *120*

Reply:
Increased to 120.